From a3a8c6f64f6ec3665048221bd1dbf5cfce2e108c Mon Sep 17 00:00:00 2001
From: Victor Wan <victor.wan@amlogic.com>
Date: Tue, 12 Aug 2014 09:18:19 +0800
Subject: [PATCH 4692/5965] defconfigs: clean up meson8_defconfig,
 meson8b_defconfig

Don't modify arch/arm/configs/meson*_defconfig by hand.

Here is the right flow:

make mesonX_defconfig
make menuconfig
<add or remove some configs>
make savedefconfig
cp defconfig arch/arm/configs/mesonX_defconfig
---
 arch/arm/configs/meson8_defconfig  |  21 +----
 arch/arm/configs/meson8b_defconfig | 122 ++++++++++++-----------------
 2 files changed, 53 insertions(+), 90 deletions(-)

diff --git a/arch/arm/configs/meson8_defconfig b/arch/arm/configs/meson8_defconfig
index da9e7c3f22a6..520be2d5aad1 100755
--- a/arch/arm/configs/meson8_defconfig
+++ b/arch/arm/configs/meson8_defconfig
@@ -17,6 +17,7 @@ CONFIG_MODULES=y
 CONFIG_MODULE_UNLOAD=y
 CONFIG_MESON_SUSPEND=y
 CONFIG_MESON_LEGACY_REGISTER_API=y
+CONFIG_GPIO_TEST=y
 CONFIG_ARM_ERRATA_751472=y
 CONFIG_ARM_ERRATA_754322=y
 CONFIG_ARM_ERRATA_764369=y
@@ -148,7 +149,6 @@ CONFIG_BT_BNEP=y
 CONFIG_BT_BNEP_MC_FILTER=y
 CONFIG_BT_BNEP_PROTO_FILTER=y
 CONFIG_BT_HIDP=y
-CONFIG_BT_RTKBTUSB=m
 CONFIG_BT_HCIUART=y
 CONFIG_BT_HCIUART_H4=y
 CONFIG_CFG80211=y
@@ -161,7 +161,6 @@ CONFIG_NEW_AM_REMOTE=y
 CONFIG_NEW_AM_IR_TX=y
 CONFIG_ADC_KEYPADS_AM=y
 CONFIG_MESON_INPUT_TOUCHSCREEN=y
-CONFIG_AML_TOUCH_ALGORITHM_SUPPORT=m
 CONFIG_FOCALTECH_CAPACITIVE_TOUCHSCREEN=y
 CONFIG_GOODIX_GT81XX_CAPACITIVE_TOUCHSCREEN=y
 CONFIG_GSLX680_COMPATIBLE_CAPACITIVE_TOUCHSCREEN=y
@@ -222,17 +221,18 @@ CONFIG_BCMDHD_POWER_OFF_IN_SUSPEND=y
 CONFIG_RTL8188EU=m
 CONFIG_RTL8188EU_MP=m
 CONFIG_BT_DEVICE=y
+CONFIG_BT_RTKBTUSB=m
 CONFIG_VSYNC_RDMA=y
 CONFIG_AM_VIDEO=y
 CONFIG_AM_ENCODER=y
 CONFIG_AM_JPEG_ENCODER=y
 CONFIG_AM_TV_OUTPUT2=y
 CONFIG_FB_AM=y
+CONFIG_FB_OSD2_CURSOR=y
 CONFIG_FB_OSD2_DEFAULT_WIDTH=32
 CONFIG_FB_OSD2_DEFAULT_HEIGHT=32
 CONFIG_FB_OSD2_DEFAULT_WIDTH_VIRTUAL=32
 CONFIG_FB_OSD2_DEFAULT_HEIGHT_VIRTUAL=32
-CONFIG_FB_OSD2_CURSOR=y
 CONFIG_AM_FB_EXT=y
 CONFIG_AM_LOGO=y
 CONFIG_AM_HDMI_ONLY=y
@@ -370,18 +370,3 @@ CONFIG_CRYPTO_PCBC=y
 CONFIG_CRYPTO_TWOFISH=y
 CONFIG_CRC_T10DIF=y
 CONFIG_CRC7=y
-CONFIG_GPIO_TEST=y	
-CONFIG_MD=y
-CONFIG_BLK_DEV_DM=y
-CONFIG_DM_DEBUG=y
-CONFIG_DM_CRYPT=y
-CONFIG_DM_UEVENT=y
-CONFIG_CRYPTO_AEAD=y
-CONFIG_CRYPTO_AUTHENC=y
-CONFIG_CRYPTO_CBC=y
-CONFIG_CRYPTO_PCBC=y
-CONFIG_CRYPTO_HMAC=y
-CONFIG_CRYPTO_MD5=y
-CONFIG_CRYPTO_DES=y
-CONFIG_CRYPTO_TWOFISH=y
-CONFIG_CRYPTO_TWOFISH_COMMON=y
diff --git a/arch/arm/configs/meson8b_defconfig b/arch/arm/configs/meson8b_defconfig
index 08a91856c2db..2bfbeddc3a03 100755
--- a/arch/arm/configs/meson8b_defconfig
+++ b/arch/arm/configs/meson8b_defconfig
@@ -164,12 +164,38 @@ CONFIG_MESON_INPUT_TOUCHSCREEN=y
 CONFIG_FOCALTECH_CAPACITIVE_TOUCHSCREEN=y
 CONFIG_GOODIX_GT81XX_CAPACITIVE_TOUCHSCREEN=y
 CONFIG_GSLX680_COMPATIBLE_CAPACITIVE_TOUCHSCREEN=y
-CONFIG_AML_TOUCH_ALGORITHM_SUPPORT=m
 CONFIG_GOODIX_GT9XX_CAPACITIVE_TOUCHSCREEN=y
 CONFIG_NOVATEK_CAPACITIVE_TOUCHSCREEN=y
 CONFIG_TOUCHSCREEN_CT36X=y
+CONFIG_SENSOR_DEVICES=y
+CONFIG_GRAVITY_BMA250=y
+CONFIG_GRAVITY_BMA222=y
+CONFIG_GRAVITY_MMA7660=y
+CONFIG_GRAVITY_MIR3DA=y
+CONFIG_GRAVITY_LIS3DH=y
+CONFIG_GRAVITY_MMA8452=y
+CONFIG_GRAVITY_MC32X0=y
+CONFIG_GRAVITY_DMARD06=y
+CONFIG_GRAVITY_MMA865X=y
+CONFIG_GRAVITY_LSM303D=y
+CONFIG_GRAVITY_DMARD10=y
+CONFIG_GRAVITY_KXTJ9=y
+CONFIG_GRAVITY_STK8313=y
+CONFIG_GRAVITY_STK8312=y
+CONFIG_GRAVITY_MM3A310=y
+CONFIG_GRAVITY_MXC622X=y
+CONFIG_GRAVITY_MXC6255XC=y
+CONFIG_LIGHT_CM36283=y
+CONFIG_LIGHT_CM3232=y
+CONFIG_LIGHT_CM3217=y
+CONFIG_LIGHT_LTR501=y
+CONFIG_LIGHT_EPL6814=y
+CONFIG_LIGHT_ISL29023=y
+CONFIG_LIGHT_STK220X=y
+CONFIG_LIGHT_LTR558=y
 CONFIG_AML_GPIO_KEY=y
 CONFIG_AML_PMU_ALGORITHM_SUPPORT=m
+CONFIG_MESON_CS_DCDC_REGULATOR=y
 CONFIG_AML_PMU=y
 CONFIG_AML1216=y
 CONFIG_AML1218=y
@@ -179,9 +205,9 @@ CONFIG_EFUSE=y
 CONFIG_AMLOGIC_THERMAL=y
 CONFIG_AMLOGIC_USB=y
 CONFIG_MMC_AML=y
+CONFIG_AML_NEXT_GEN_NAND=y
 CONFIG_AML_NAND_KEY=y
 # CONFIG_AM_NAND is not set
-CONFIG_AML_NEXT_GEN_NAND=y
 CONFIG_AM_ETHERNET=y
 CONFIG_AML_PHY=y
 CONFIG_AML_LAN8720=y
@@ -193,58 +219,41 @@ CONFIG_AM_WIFI_SD_MMC=y
 CONFIG_AM_WIFI_USB=y
 CONFIG_SDIO_DHD_CDC_WIFI_40181_MODULE=m
 CONFIG_RTL8188EU=m
+CONFIG_RTL8188EU_MP=m
 CONFIG_RTL8189ES=m
 CONFIG_RTL8723BS=m
-CONFIG_RTL8188EU_MP=m
 CONFIG_BT_DEVICE=y
 CONFIG_VSYNC_RDMA=y
 CONFIG_AM_VIDEO=y
 CONFIG_AM_ENCODER=y
 CONFIG_AM_JPEG_ENCODER=y
 CONFIG_FB_AM=y
+CONFIG_FB_OSD2_CURSOR=y
 CONFIG_FB_OSD2_DEFAULT_WIDTH=32
 CONFIG_FB_OSD2_DEFAULT_HEIGHT=32
 CONFIG_FB_OSD2_DEFAULT_WIDTH_VIRTUAL=32
 CONFIG_FB_OSD2_DEFAULT_HEIGHT_VIRTUAL=32
-CONFIG_FB_OSD2_CURSOR=y
-CONFIG_AMLOGIC_BACKLIGHT=y
-CONFIG_AM_GE2D=y
 CONFIG_AM_LOGO=y
+CONFIG_AM_HDMI_ONLY=y
+CONFIG_VIUIN=y
 CONFIG_POST_PROCESS_MANAGER=y
 CONFIG_POST_PROCESS_MANAGER_PPSCALER=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE=y
+CONFIG_AMLOGIC_CAPTURE_FRAME_ROTATE=y
+CONFIG_AMLOGIC_VM_DISABLE_VIDEOLAYER=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_GC0308=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_GC0328=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_GC2035=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_SP0838=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_SP2518=y
 CONFIG_V4L_AMLOGIC_VIDEO=y
 CONFIG_V4L_AMLOGIC_VIDEO2=y
 CONFIG_AMLOGIC_IONVIDEO=y
-CONFIG_SENSOR_DEVICES=y
-CONFIG_GRAVITY_BMA250=y
-CONFIG_GRAVITY_BMA222=y
-CONFIG_GRAVITY_MMA7660=y
-CONFIG_GRAVITY_MIR3DA=y
-CONFIG_GRAVITY_LIS3DH=y
-CONFIG_GRAVITY_MMA8452=y
-CONFIG_GRAVITY_MC32X0=y
-CONFIG_GRAVITY_DMARD06=y
-CONFIG_GRAVITY_MMA865X=y
-CONFIG_GRAVITY_LSM303D=y
-CONFIG_GRAVITY_DMARD10=y
-CONFIG_GRAVITY_KXTJ9=y
-CONFIG_GRAVITY_STK8312=y
-CONFIG_GRAVITY_STK8313=y
-CONFIG_GRAVITY_MM3A310=y
-CONFIG_GRAVITY_MXC622X=y
-CONFIG_GRAVITY_MXC6255XC=y
-CONFIG_LIGHT_CM36283=y
-CONFIG_LIGHT_CM3232=y
-CONFIG_LIGHT_CM3217=y
-CONFIG_LIGHT_LTR501=y
-CONFIG_LIGHT_EPL6814=y
-CONFIG_LIGHT_ISL29023=y
-CONFIG_LIGHT_STK220X=y
-CONFIG_LIGHT_LTR558=y
-CONFIG_AM_HDMI_ONLY=y
 CONFIG_DEINTERLACE=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
+CONFIG_CMA=y
+CONFIG_CMA_SIZE_MBYTES=8
 CONFIG_MTD=y
 CONFIG_MTD_BLOCK=y
 CONFIG_PROC_DEVICETREE=y
@@ -254,6 +263,11 @@ CONFIG_SCSI=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_BLK_DEV_SR=y
 CONFIG_SCSI_MULTI_LUN=y
+CONFIG_MD=y
+CONFIG_BLK_DEV_DM=y
+CONFIG_DM_DEBUG=y
+CONFIG_DM_CRYPT=y
+CONFIG_DM_UEVENT=y
 CONFIG_NETDEVICES=y
 CONFIG_TUN=y
 CONFIG_PPP=y
@@ -274,19 +288,14 @@ CONFIG_INPUT_MISC=y
 CONFIG_INPUT_UINPUT=y
 CONFIG_DEVPTS_MULTIPLE_INSTANCES=y
 CONFIG_SPI=y
-CONFIG_MEDIA_SUPPORT=y
 CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_MEDIA_USB_SUPPORT=y
 CONFIG_USB_VIDEO_CLASS=y
 CONFIG_V4L_PLATFORM_DRIVERS=y
 CONFIG_BACKLIGHT_LCD_SUPPORT=y
-CONFIG_SOUND=y
-CONFIG_SND=y
-CONFIG_SND_SOC=y
+CONFIG_SND_USB_AUDIO=y
 CONFIG_SND_AML_M8_SOC=y
 CONFIG_SND_AML_M8=y
-CONFIG_SND_USB=y
-CONFIG_SND_USB_AUDIO=y
 CONFIG_UHID=y
 CONFIG_USB=y
 CONFIG_USB_ACM=y
@@ -329,7 +338,6 @@ CONFIG_UDF_FS=y
 CONFIG_MSDOS_FS=y
 CONFIG_VFAT_FS=y
 CONFIG_TMPFS=y
-CONFIG_CONFIGFS_FS=y
 CONFIG_CRAMFS=y
 CONFIG_CIFS=y
 CONFIG_NLS_CODEPAGE_437=y
@@ -350,37 +358,7 @@ CONFIG_DEBUG_INFO=y
 CONFIG_SECURITY=y
 CONFIG_SECURITY_NETWORK=y
 CONFIG_SECURITY_SELINUX=y
-CONFIG_CRC_T10DIF=y
-CONFIG_CRC7=y
-CONFIG_AM_LCD_OUTPUT=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE=y
-CONFIG_AMLOGIC_CAPTURE_FRAME_ROTATE=y
-CONFIG_AMLOGIC_VM_DISABLE_VIDEOLAYER=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_GC0308=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_GC0328=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_SP0838=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_GC2035=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_SP2518=y
-CONFIG_CMA=y
-CONFIG_CMA_SIZE_MBYTES=8
-CONFIG_MESON_CS_DCDC_REGULATOR=y
-CONFIG_AML_DVFS=y
-CONFIG_TVIN=y
-CONFIG_TVIN_VDIN=y
-CONFIG_VIUIN=y
-CONFIG_V4L_AMLOGIC_VIDEO2=y
-CONFIG_AM_ENCODER=y
-CONFIG_MD=y
-CONFIG_BLK_DEV_DM=y
-CONFIG_DM_DEBUG=y
-CONFIG_DM_CRYPT=y
-CONFIG_DM_UEVENT=y
-CONFIG_CRYPTO_AEAD=y
-CONFIG_CRYPTO_AUTHENC=y
-CONFIG_CRYPTO_CBC=y
 CONFIG_CRYPTO_PCBC=y
-CONFIG_CRYPTO_HMAC=y
-CONFIG_CRYPTO_MD5=y
-CONFIG_CRYPTO_DES=y
 CONFIG_CRYPTO_TWOFISH=y
-CONFIG_CRYPTO_TWOFISH_COMMON=y
+CONFIG_CRC_T10DIF=y
+CONFIG_CRC7=y
-- 
2.19.0

